### PR TITLE
installation guide for redis on developer's personal computer

### PR DESCRIPTION
**Pull request by anglofox**

### CODE REVIEW

**Comment by anglofox** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;KHzf7VNz&#x2F;555-optimization-redis-caching"> Optimization: Redis Caching</a>

File: redis-installation-guide.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+#### Configure Redis
+To start off, create a configuration directory. We will use the conventional /etc/redis directory. 
+
+```sh
+$ sudo mkdir /etc/redis
+```
+Now, copy over the [Redis configuration file][googledoc] that is in the FINDMINE Engineering Folder on Google Drive
+
+```sh
+$ sudo cp /path/to/Mishki/Engineering/Docs/redis.conf /etc/redis
```
Comment (trivigy) [inline redis-installation-guide.md]: Can you change these folder references to something like /path/to/install/ or something? This way we'll be able to keep this tutorial in this public repo.

**Comment by trivigy** [comment]
I would also point out that this tutorial was built based on the ubuntu 16.04 machine. You do make a reference to the digital ocean tutorial and so I am not sure if this is really necessary. Your call.

**Comment by trivigy** [comment]
Wh

**Comment by trivigy** [comment]
Ooops... closed the pull request by mistake and then reopened. Anyway, as soon as you make these changes I'll approve the pull request. This looks fantastic. Great work.

**Comment by anglofox** [comment]
Hey,

I have updated the doc as per your request!

Best,

**Pull request merged**